internal/controller/services/monitoring/resources/monitoring-stack.tmpl.yaml

```
@@ -0,0 +1,25 @@
+metadata:
+  name: {{ .MonitoringStackName }}
+  namespace: {{ .Monitoring.Namespace }}
+spec:
+  alertmanagerConfig:
+    disabled: false
+  logLevel: debug
+  prometheusConfig:
+    enableOtlpHttpReceiver: true
+    persistentVolumeClaim:
+      resources:
+        requests: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)